mypy/myunit/__init__.py

```diff
@@ -293,6 +293,7 @@ def run_single_test(name: str, test: Any) -> Tuple[bool, bool]:
         test.run()
     except BaseException as e:
         if isinstance(e, KeyboardInterrupt):
+            test.tear_down()
             raise
         exc_type, exc_value, exc_traceback = sys.exc_info()
     test.tear_down()  # FIX: check exceptions
```

Comment:
How about let's move this line into a `finally` block? Then we only have to say `tear_down` in one place, and it's easier to see that it'll always be called.